### PR TITLE
fix nolint syntax

### PR DESCRIPTION
**Pull request by chlebowa**
https://github.com/insightsengineering/teal.modules.general/pull/636#issuecomment-1952150583

### CODE REVIEW

File: R/tm_g_distribution.R [hunk 1]
```diff
@@ -560,7 +560,7 @@ srv_distribution <- function(id,
         qenv <- teal.code::eval_code(
           qenv,
           substitute(
-            expr = ANL[[g_var]] <- forcats::fct_na_value_to_level(as.factor(ANL[[g_var]]), "NA"), # nolint object_name_linter
+            expr = ANL[[g_var]] <- forcats::fct_na_value_to_level(as.factor(ANL[[g_var]]), "NA"), # nolint: object_name_linter, line_length_linter.
```
Comment (averissimo) [inline R/tm_g_distribution.R:563]: Linter inception! It seems like we can commit the `_linter` suffix to make it smaller and it barely fits on 120 words

```suggestion
            expr = ANL[[g_var]] <- forcats::fct_na_value_to_level(as.factor(ANL[[g_var]]), "NA"), # nolint: object_name.
```

File: R/tm_g_distribution.R [hunk 2]
```diff
@@ -576,7 +576,7 @@ srv_distribution <- function(id,
         qenv <- teal.code::eval_code(
           qenv,
           substitute(
-            expr = ANL[[s_var]] <- forcats::fct_na_value_to_level(as.factor(ANL[[s_var]]), "NA"), # nolint object_name_linter
+            expr = ANL[[s_var]] <- forcats::fct_na_value_to_level(as.factor(ANL[[s_var]]), "NA"), # nolint: object_name_linter, line_length_linter.
```
Comment (averissimo) [inline R/tm_g_distribution.R:579]:
```suggestion
            expr = ANL[[s_var]] <- forcats::fct_na_value_to_level(as.factor(ANL[[s_var]]), "NA"), # nolint: object_name.
```

**Review by averissimo — APPROVED** [review]
Looks good!

It seems we can omit the `_linter` suffix and barely avoid the line linter.

Add that if you wish.

**Comment by chlebowa** [comment]
I removed the suffixes and spotted two more syntax errors I had missed.

**Pull request merged**